boards/arm/nucleo_wba55cg/doc/nucleo_wba55cg.rst

```diff
@@ -0,0 +1,292 @@
+
+- ECOPACK2 compliant package
+
+More information about STM32WBA series can be found here:
+
+- `STM32WBA Series on www.st.com`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_wba55cg board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+| RNG       | on-chip    | True Random number generator        |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_wba55cg/nucleo_wba55cg_defconfig``
+
+Connections and IOs
+===================
+
+Nucleo WBA55CG Board has 4 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- USART_1 TX/RX : PB12/PA8
+- I2C_1_SCL : PB2
+- I2C_1_SDA : PB1
+- USER_PB : PC13
+- LD1 : PB4
+- SPI_1_NSS : PA12 (arduino_spi)
+- SPI_1_SCK : PB4 (arduino_spi)
+- SPI_1_MISO : PB3 (arduino_spi)
+- SPI_1_MOSI : PA15 (arduino_spi)
+
+System Clock
+------------
+
+Nucleo WBA55CG System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by HSE+PLL clock at 100MHz.
+
+Serial Port
+-----------
+
+Nucleo WBA55CG board has 1 U(S)ARTs. The Zephyr console output is assigned to USART1.
+Default settings are 115500 8N1.
+
+
+Programming and Debugging
+*************************
+
+Nucleo WBA55CG board includes an ST-LINK/V3 embedded debug tool interface.
+It could be used for flash and debug using either OpenOCD or STM32Cube ecosystem tools.
```

Comment:
It could be used for flashing and debugging using either OpenOCD or STM32Cube ecosystem tools.